src/components/visualizations/chart/highcharts/plugins/dualAxesLabelFormatter.ts

```diff
@@ -124,8 +126,11 @@ export function dualAxesLabelFormatter() {
     this.value = formatLabel(this.value, tickPositions);
 
     const stackMeasuresToPercent = get(this, "chart.userOptions.stackMeasuresToPercent", false);
-    const seriesInAxis = get(this, "axis.series", []).length;
-    if (stackMeasuresToPercent && seriesInAxis > 0) {
+    const series = get(this, "axis.series", []);
+    const isLineChartType = series.some(
+        (seriesItem: ISeriesItem) => seriesItem.type === VisualizationTypes.LINE,
```

Comment:
it could be `(seriesItem: ISeriesItem) => isLineChart(seriesItem.type)`

Reply:
fixed